internal/testrunner/runners/system/runner.go

```diff
@@ -847,6 +851,19 @@ func (r *runner) prepareScenario(ctx context.Context, config *testConfig, svcInf
 		return nil, fmt.Errorf("reading data stream manifest failed: %w", err)
 	}
 
+	// Temporarily until independent Elastic Agents are enabled by default,
+	// enable independent Elastic Agents if package defines that requires root privileges
+	if scenario.pkgManifest.Agent.Privileges.Root || scenario.dataStreamManifest.Agent.Privileges.Root {
+		r.options.RunIndependentElasticAgent = true
+	}
+
+	// If the environment variable is present, it always has preference over the root
+	// privileges value (if any) defined in the manifest file
+	v, ok := os.LookupEnv(enableIndependentAgents)
+	if ok {
+		r.options.RunIndependentElasticAgent = strings.ToLower(v) == "true"
+	}
```

Comment:
It needs to be checked again the environment variable, to ensure it has preference over the value in the manifest files.